rslib/src/search/note_original_position_order.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS sort_order;
+CREATE TEMPORARY TABLE sort_order (
+  pos integer PRIMARY KEY,
+  nid integer NOT NULL UNIQUE
+);
+INSERT INTO sort_order (nid)
+SELECT nid
+FROM cards
+GROUP BY nid
+ORDER BY COALESCE(odue, due);
```

Comment:
This looks incorrect? It's sorting cards based on their due date, not the original position.